rest/route/host_spawn.go

```diff
@@ -227,37 +205,49 @@ func checkInstanceTypeHostStopped(h *host.Host) error {
 	return nil
 }
 
+func checkExpirableHostLimitExceeded(userId string) error {
```

Comment:
Just double checking - did you mean to include changes to forever hosts here?

Reply:
Yeah sorry to mix things--because I moved options into something the user passes in the body, this allows you to extend the expiration using this host modify (which seems fine to me) but then this should also check expirable limits.